Docs/Configuration.tex

```diff
@@ -3290,33 +3315,33 @@ \subsection{Debug Properties}\label{miscdebugprops}
   \texttt{AppleDebug}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Enable \texttt{boot.efi} debug log saving to OpenCore log.
+  \textbf{Description}: Enable writing the \texttt{boot.efi} debug log to the OpenCore log.
 
   \emph{Note}: This option only applies to 10.15.4 and newer.
 
 \item
   \texttt{ApplePanic}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Save macOS kernel panic to OpenCore root partition.
+  \textbf{Description}: Save macOS kernel panic output to the OpenCore root partition.
 
   The file is saved as \texttt{panic-YYYY-MM-DD-HHMMSS.txt}. It is strongly
-  recommended to have \texttt{keepsyms=1} boot argument to see debug symbols
-  in the panic log. In case it was not present \texttt{kpdescribe.sh} utility
-  (bundled with OpenCore) may be used to partially recover the stacktrace.
+  recommended to set the \texttt{keepsyms=1} boot argument to see debug symbols
+  in the panic log. In case where it is not present, the \texttt{kpdescribe.sh}
+  utility (bundled with OpenCore) may be used to partially recover the stacktrace.
 
-  Development and debug kernels produce more helpful kernel panics.
-  Consider downloading and installing \texttt{KernelDebugKit} from
+  Development and debug kernels produce more useful kernel panic logs.
+  Consider downloading and installing the \texttt{KernelDebugKit} from
   \href{https://developer.apple.com}{developer.apple.com} when debugging a problem.
-  To activate a development kernel the boot argument \texttt{kcsuffix=development} should be added.
-  Use \texttt{uname -a} command to ensure that the current loaded
+  To activate a development kernel, the boot argument \texttt{kcsuffix=development} should be added.
+  Use the \texttt{uname -a} command to ensure that the current loaded
   kernel is a development (or a debug) kernel.
 
-  In case OpenCore kernel panic saving mechanism was not used, kernel panics may
-  still be found in \\
-  \texttt{/Library/Logs/DiagnosticReports} directory.
-  Starting with macOS Catalina kernel panics are stored in JSON format, so they
-  need to be preprocessed before passing to \texttt{kpdescribe.sh}:
+  In cases where the OpenCore kernel panic saving mechanism is not used, kernel panic
+  logs may still be found in the \texttt{/Library/Logs/DiagnosticReports} directory.
+
+  Starting with macOS Catalina, kernel panics are stored in JSON format and
+  need to be preprocessed before being passed to \texttt{kpdescribe.sh}:
```

Comment:
I think `passing` is fine here. e.g. passing an argument to a script.

Reply:
Correct that it could be like that but "being passed to" makes it explicitly clear we are talking about passing the preprocessed item to the script.

That is, with just "passing", you could do the preprocessing, Task 1, before passing (on to running) the script, a separate and not necessarily directly related Task 2.

Reply:
Reverted to "passing" to close out